tests/test_geovector.py

```diff
@@ -515,6 +517,10 @@ def test_polygonize_line():
     assert result == expected_result
 
 
+@pytest.mark.skipif(
+    version.parse(shapely.__version__) >= version.parse('1.7b1'),
+    reason="Shapely < 1.7b1 is required",
+)
```

Comment:
I guess this silences #269, can we add a comment with a link to the issue?

Reply:
Done.